[
    {
        "content": "<p>From: Fabian Huch &lt;<a href=\"mailto:huch@in.tum.de\">huch@in.tum.de</a>&gt;<br>\nWhen retrieving local propositions inside a proof and applying standard <br>\noperations on them (e.g., Thm.unconstrainT), I get the following error <br>\n(Isabelle2023):</p>\n<p>exception CONTEXT (\"No content for theory certificate Scratch:11571\", <br>\n[], [], [\"True\"], NONE) raised (line 561 of \"thm.ML\")</p>\n<p>The problem is easy to reproduce, e.g. with the following code:</p>\n<p>proof -<br>\n   have True ..<br>\n   ML_val ‹Proof_Context.facts_of @{context} |&gt; Facts.props |&gt; map <br>\n(Thm.unconstrainT o fst)›</p>\n<p>This is probably due to compression of some certificates inside this <br>\ncontext (where full certificates are replaced by ids). Is there any way <br>\nto work around this?</p>\n<p>Fabian</p>",
        "id": 392225610,
        "sender_full_name": "Email Gateway",
        "timestamp": 1695256247
    }
]